internal/controller/pdbwatcher_controller.go

```diff
@@ -35,9 +34,9 @@ type PDBWatcherReconciler struct {
 // +kubebuilder:rbac:groups=apps.mydomain.com,resources=pdbwatchers,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=apps.mydomain.com,resources=pdbwatchers/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=apps.mydomain.com,resources=pdbwatchers/finalizers,verbs=update
-// +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;update
+// +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;update;watch
 // +kubebuilder:rbac:groups=apps,resources=statefulsets,verbs=get;list;update
-// +kubebuilder:rbac:groups=policy,resources=poddisruptionbudgets,verbs=get;list
+// +kubebuilder:rbac:groups=policy,resources=poddisruptionbudgets,verbs=get;list;create;watch
```

Comment:
nitpick these should go in commetns above your conroller so that if they're ever seperated or removed we keep a minumum